app/services/forest_liana/ability/permission/smart_action_approval.rb

```diff
@@ -0,0 +1,67 @@
+require 'jwt'
+
+module ForestLiana
+  module Ability
+    module Permission
+    class SmartActionApproval
```

Comment:
🥇 I didn't see that coming. So in here, you should use the `signedApprovalRequest` as source of truth.

Reply:
Are we only handling approvals in here? (I'm a little bit lost)

Reply:
We handle all the kind of approvals/trigger permission of a smartaction here.